libs/community/langchain_community/document_loaders/base.py

```diff
@@ -60,6 +61,19 @@ def lazy_load(
             f"{self.__class__.__name__} does not implement lazy_load()"
         )
 
+    async def aload(self) -> List[Document]:
+        """Load data into Document objects."""
+        return [doc async for doc in self.alazy_load()]
+
+    async def alazy_load(self) -> AsyncIterator[Document]:
```

Comment:
This looks great.

Would you mind adding a unit test to cover the default implementation with a fake loader that just returns a list?

We can merge after that

Reply:
done